capture.py

```diff
@@ -352,16 +352,30 @@ def apply_view(panel, **options):
         cmds.setAttr(attr, value)
 
 
-def parse_active_view():
-    """Parse the current settings from the active view"""
+def parse_active_panel():
```

Comment:
Does this simply return the currently active panel? Are we sure "parse" is the right word? Are we sure this isn't already trivial enough to write ourseves, per use, when we need it, rather than making this function?